selfdrive/car/hyundai/carstate.py

```diff
@@ -21,7 +21,9 @@ def __init__(self, CP):
     self.cruise_buttons = deque([Buttons.NONE] * PREV_BUTTON_SAMPLES, maxlen=PREV_BUTTON_SAMPLES)
     self.main_buttons = deque([Buttons.NONE] * PREV_BUTTON_SAMPLES, maxlen=PREV_BUTTON_SAMPLES)
 
-    self.gear_msg_canfd = "GEAR_ALT" if CP.flags & HyundaiFlags.CANFD_ALT_GEARS else "GEAR_SHIFTER"
+    self.gear_msg_canfd = "GEAR_ALT_2" if CP.flags & HyundaiFlags.CANFD_ALT_GEARS_2 else \
+                          "GEAR_ALT" if CP.flags & HyundaiFlags.CANFD_ALT_GEARS else \
+                          "GEAR_SHIFTER"
```

Comment:
None of the existing signals work for this car? If not, we should find two signals that work for all cars, I'd like to avoid three.

Reply:
That’d be great! Here’s the route of the 2023 Niro HEV:
- `db04d2c63990e3ba|2022-12-21--00-18-08`

Reply:
Do you have a route where the user shifts into different gears? 30 seconds to a minute of constantly switching (few seconds in between) would be ideal to find other signals

Reply:
If you want to try to find some yourself, you can use this script: https://github.com/commaai/openpilot/pull/26683 Let me know if you have any feedback!

Reply:
@deanlee this script is immensely helpful to find similar bits and signals. Can some of its functionality be baked into Cabana easily enough?

Reply:
Checked for a while, couldn't find any common gear message. Looks like three is required. It's worth thinking about any possible ways to clean this up in the future though!